src/americana.js

```diff
@@ -45,9 +45,11 @@ function buildLayers() {
     lyrBoundary.stateCasing,
     lyrBoundary.countryCasing,
 
-    lyrWater.water,
     lyrWater.waterway,
     lyrWater.waterwayIntermittent,
+    lyrWater.waterLine,
+    lyrWater.waterLineIntermittent,
+    lyrWater.water,
```

Comment:
The water fill layer needs to draw over the waterway line layer; otherwise, there will be a prominent line down the middle of the river that looks like a seam.